### PR TITLE
CNF-12833: PTP CI: 4.16 and 4.17 linuxptp daemon deployment not found/failed

### PR DESCRIPTION
**Pull request by nishant-parekh**
update authorization to access CI registry for 4.16 and 4.17 in the podman container.

  * Add env variable to pass release name to podman container.
  * Add check to use appropriate authorization to access registry

### CODE REVIEW

**Comment by nishant-parekh** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-ptp

File: ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh [hunk 1]
```diff
@@ -68,8 +71,16 @@ spec:
           # set +x here to hide pass from log
           set +xe
           echo "podman login with serviceaccount"
-          pass=$( jq .\"image-registry.openshift-image-registry.svc:5000\".password /var/run/secrets/openshift.io/push/.dockercfg )
-          podman login -u serviceaccount -p ${pass:1:-1} image-registry.openshift-image-registry.svc:5000 --tls-verify=false
+
+          if [[ "$T5CI_VERSION" == "4.16" || "$T5CI_VERSION" == "4.17"]];
```
Comment (jzding) [inline ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh:75]: I would check `if version > 4.15 instead`. That way we don't have to update this with every future release.

Comment (nishant-parekh) [reply ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh]: I didnt want to use the release version in this logic so I have updated to try the old way first and using a try catch block to detect failure and use the logic as a catch for newer releases.

**Comment by nishant-parekh** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-ptp

**Comment by nishant-parekh** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-ptp

**Comment by nishant-parekh** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-ptp

**Comment by nishant-parekh** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-ptp

**Comment by nishant-parekh** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.14-e2e-telco5g-ptp

**Pull request merged**